examples_tests/49.ComputeFFT/fft_convolve_ifft.comp

```diff
@@ -0,0 +1,96 @@
+// WorkGroup Size
+
+#define _NBL_GLSL_WORKGROUP_SIZE_ 256
+
+layout(local_size_x=_NBL_GLSL_WORKGROUP_SIZE_, local_size_y=1, local_size_z=1) in;
+
+ 
+#define _NBL_GLSL_EXT_FFT_GET_PARAMETERS_DEFINED_
+#define _NBL_GLSL_EXT_FFT_GET_DATA_DEFINED_
+#define _NBL_GLSL_EXT_FFT_SET_DATA_DEFINED_
+#define _NBL_GLSL_EXT_FFT_GET_PADDED_DATA_DEFINED_
+#include "nbl/builtin/glsl/ext/FFT/fft.glsl"
+
+// Input Descriptor
+
+layout(set=0, binding=0) readonly restrict buffer InputBuffer
+{
+	nbl_glsl_complex inData[];
+};
+
+layout(set=0, binding=1) restrict readonly buffer KernelBuffer
+{
+	nbl_glsl_complex kerData[];
+};
+
+// Output Descriptor
+
+layout(set=0, binding=2) restrict buffer OutputBuffer
+{
+	nbl_glsl_complex outData[];
+};
```

Comment:
if you have the same invocation responsible for each element and use `barrier` to sync the entire axis, then you can just use a single SSBO as both input and output